### DISCUSSION TITLE
Wiki width too narrow

### DISCUSSION THREAD

**Discussion by ScottLangham** [community/community#120861] [Repositories]
### Select Topic Area

Product Feedback

### Body

I've added some content to the Wiki in my repo that includes a table with just 4 columns.

When I attempt to view it, the Wiki only displays in a thin vertical box down the center of my browser. This means I cannot see much of this table and have to scroll, but to be honest this makes it unusable as I need to see more of the row to know which row I'm looking at.

I have loads of screen space if only the wiki display could use it.

It looks like a lot of people have the same issue going back to 2015. Please see: https://github.com/isaacs/github/issues/346

**Comment by Cueball** [comment]
If you're not averse to using a 3rd-party userscript, Wide GitHub should do what you want; I've been using it for nearly a year to do what you're asking. Repo is at:

https://github.com/xthexder/wide-github
